geode-core/src/distributedTest/java/org/apache/geode/cache30/ReconnectWithCacheXMLDUnitTest.java

```diff
@@ -55,10 +61,22 @@
   @Rule
   public DistributedRestoreSystemProperties restoreSystemProperties =
       new DistributedRestoreSystemProperties();
+  private int locatorPort;
 
   @Override
   public final void postSetUp() {
     oldPropertySetting = System.setProperty(xmlProperty, "true");
+    // stress testing needs this so that join attempts don't give up too soon
+    Invoke.invokeInEveryVM(() -> System.setProperty("p2p.joinTimeout", "120000"));
+    // reconnect tests should create their own locator so as to not impact other tests
+    VM locatorVM = VM.getVM(0);
+    final int port = locatorVM.invoke(() -> {
+      System.setProperty(BYPASS_DISCOVERY_PROPERTY, "true");
```

Comment:
Why does this test need to set this system property (bypass discovery)?

Reply:
It cuts 10-15 seconds off the time this test needs to run